pkgs/tools/games/minecraft/minecraft-server-hibernation/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "minecraft-server-hibernation";
+  version = "2.4.0";
```

Comment:
Latest tagged version on github repo is [2.4.9](https://github.com/gekware/minecraft-server-hibernation/tree/v2.4.9)

Reply:
v2.4.9 is marked as a pre-release, but I tested it and everything seems to work fine so I'll update it.